data/core/game_balance.json

```diff
@@ -16,16 +16,30 @@
   {
     "type": "EXTERNAL_OPTION",
     "name": "PLAYER_MAX_STAMINA",
-    "info": "Sets max stamina value of the player.",
+    "info": "Sets the base max stamina value of the player, before cardio modifiers.",
     "stype": "int",
-    "value": 10000
+    "value": 2000
```

Comment:
this actually affects spellcasting in a huge way. all magiclysm spells that use stamina need to be scaled accordingly if this is changed.